include/v8.h

```diff
@@ -102,6 +102,7 @@ static bool HasDisabledFeatures();
 
 static void Print(const char* format, ...);
 static void Diagnostic(const char* format, ...);
+static void CommandDiagnostic(const char* format, ...);
```

Comment:
@bhackett1024 I made one more change that sets it further apart from `Print`: I added a prefix to its messages, since we want to easily find these in the log - https://github.com/replayio/chromium-v8/pull/182/commits/03d33f3d4e41e4e5034e337c704e2b208b667765